src/wp-includes/class-wp-scripts.php

```diff
@@ -390,8 +390,18 @@ public function do_item( $handle, $group = false ) {
 			return true;
 		}
 
+		$strategy = $this->get_eligible_loading_strategy( $handle );
+		if( '' !== $strategy ) {
+			$strategy = ' '.$strategy;
+		}
 		$tag  = $translations . $cond_before . $before_handle;
-		$tag .= sprintf( "<script%s src='%s' id='%s-js'></script>\n", $this->type_attr, $src, esc_attr( $handle ) );
+		$tag .= sprintf( 
+			"<script%s src='%s' id='%s-js'%s></script>\n", 
+			$this->type_attr, 
+			esc_url( $src ), 
+			esc_attr( $handle ), 
+			$strategy 
```

Comment:
Looks like the numbered placeholders got removed again.

Reply:
@joemcgill  It was breaking as the $s was getting considered as a variable. 

I'll either have to swap double and single quotes, but it will break almost every test case in the test file. Or use escape literals before the current single quote, I tried it, but the string looks more confusing after that.